src/cloud/project/magento-env-yaml.md

```diff
@@ -67,3 +67,45 @@ When a failure occurs because of an unexpected value in the `.magento.env.yaml`
 ```
 
 Make any corrections, commit, and push the changes. If you do not receive an error message, then the changes to your configuration file pass the validation.
+
+## Create configuration file from CLI
+
+You can use the following `ece-tools` CLI commands to generate a `.magento.env.yaml` configuration file for a Cloud environment.
```

Comment:
May be better to lead with generate. "You can generate a ... using the following ...commands:"